plugins/com.google.cloud.tools.eclipse.appengine.deploy/src-test/com/google/cloud/tools/eclipse/appengine/deploy/AppEngineDeployOutputTest.java

```diff
@@ -25,7 +25,7 @@
 public class AppEngineDeployOutputTest {
 
   @Rule public ExpectedException thrown = ExpectedException.none();
-
+  
```

Comment:
I can't make this test fail, eithe rin Eclipse or at the command line. It appears to be picking up an older version.